Dependencies/UrlLib/Source/Apple/UrlRequest.mm

```diff
@@ -44,6 +44,10 @@ void ResponseType(UrlResponseType value)
             if ([scheme isEqual:@"app"])
             {
                 NSString* path{[[NSBundle mainBundle] pathForResource:url.path ofType:nil]};
+                if (path == nil)
+                {
+                    throw std::runtime_error{"Invalid resource path in app bundle."};
```

Comment:
Does this string end up on the JS side? If so, can we put the path in the error?

Reply:
Not this one but the other throw is. When throwing, the reason is not forwarded to JS. JS handles the exception and display its own error message. But you get the JS callstack so it's pretty easy to figure out what's causing the exception.